src/app/Components/Sidebar/Sidebar.tsx

```diff
@@ -27,9 +27,9 @@ class Sidebar extends Component {
                     <SidebarNav Icon={TradingViewLogo} name="Trading View" link="/backtesting" />
                 </div>
                 <div className="flex-column sidebar-column" style={{ justifyContent: 'flex-end' }}>
-                    <SidebarLink Icon={Coffee} name="Donate" onClick={() => window.mainPreload.general.openLink('https://www.buymeacoffee.com/ColtonS')} />
+                    <SidebarLink Icon={Coffee} name="Donate" onClick={() => window.ThreeCPM.Repository.General.openLink('https://www.buymeacoffee.com/ColtonS')} />
```

Comment:
We have to find a different trick for those ones. The repositories are gateways toward data, not "features". I propose to use a dedicated Link/Button component, inspired by what I did there: https://github.com/coltoneshaw/3c-portfolio-manager/blob/newTypes/src/app/Pages/Settings/Components/FeedbackOrBugButton.tsx

Reply:
That makes sense, I wasn't thinking that when I made it. I'll see if I can implement it with custom link logic based on electron / not

Reply:
Finished - Created an open link function to pass the data into.